<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XpcEditorComp</class>
 <widget class="QWidget" name="XpcEditorComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Configuration files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_cf">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
     show-decoration-selected: 1; /* make the selection span the entire width of the view */
 }

 QListView::item {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FAFBFE, stop: 1 #DCDEF1);
 }

 QListView::item:alternate {
     background: #EEEEEE;
 }

 QListView::item:selected {
     border: 1px solid #6a6ea9;
 }

 QListView::item:selected:!active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ABAFE5, stop: 1 #8588B2);
 }

 QListView::item:selected:active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #6a6ea9, stop: 1 #888dd9);
 }
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::TargetMoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_add_cf">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Add.svg</normaloff>:/Icons/Add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_remove_cf">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Delete.svg</normaloff>:/Icons/Delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEdit_cf"/>
     </item>
     <item>
      <widget class="QToolButton" name="buttonEdit_cf">
       <property name="toolTip">
        <string comment="File selection with a dialog window">File selection</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Open.svg</normaloff>:/Icons/Open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Package directories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_pd">
     <property name="styleSheet">
      <string notr="true">QListView {
     show-decoration-selected: 1; /* make the selection span the entire width of the view */
 }

 QListView::item {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FAFBFE, stop: 1 #DCDEF1);
 }

 QListView::item:alternate {
     background: #EEEEEE;
 }

 QListView::item:selected {
     border: 1px solid #6a6ea9;
 }

 QListView::item:selected:!active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ABAFE5, stop: 1 #8588B2);
 }

 QListView::item:selected:active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #6a6ea9, stop: 1 #888dd9);
 }
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::TargetMoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_add_pd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Add.svg</normaloff>:/Icons/Add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_remove_pd">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Delete.svg</normaloff>:/Icons/Delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="lineEdit_pd"/>
     </item>
     <item>
      <widget class="QToolButton" name="buttonEdit_pd">
       <property name="toolTip">
        <string comment="File selection with a dialog window">File selection</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Open.svg</normaloff>:/Icons/Open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Packages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_pk">
     <property name="styleSheet">
      <string notr="true">QListView {
     show-decoration-selected: 1; /* make the selection span the entire width of the view */
 }

 QListView::item {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #FAFBFE, stop: 1 #DCDEF1);
 }

 QListView::item:alternate {
     background: #EEEEEE;
 }

 QListView::item:selected {
     border: 1px solid #6a6ea9;
 }

 QListView::item:selected:!active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ABAFE5, stop: 1 #8588B2);
 }

 QListView::item:selected:active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #6a6ea9, stop: 1 #888dd9);
 }
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::TargetMoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_add_pk">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Add.svg</normaloff>:/Icons/Add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_remove_pk">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Delete.svg</normaloff>:/Icons/Delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="lineEdit_pk"/>
     </item>
     <item>
      <widget class="QToolButton" name="buttonEdit_pk">
       <property name="toolTip">
        <string comment="File selection with a dialog window">File selection</string>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Open.svg</normaloff>:/Icons/Open.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../iqtgui/iqtgui.qrc"/>
 </resources>
 <connections/>
</ui>
